mne/annotations.py

```diff
@@ -0,0 +1,37 @@
+# Authors: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from datetime import datetime
+import time
+
+
+class Annotations():
+
+    def __init__(self, orig_time, segments, descriptions):
+        """Annotation object for annotating segments of raw data.
+
+        Parameters
+        ----------
+        orig_time: int | instance of datetime
+            Timestamp or a datetime determining the starting time of segment
+            acquisition.
+        segments: array
+            Array that contains segments of data in shape (t_start, duration).
```

Comment:
I would use 2 arrays. One called onsets and one called durations

shape of array should be explicit in docstring